client/src/components/NewLocationReview.jsx

```diff
@@ -0,0 +1,275 @@
+import { inputRef, useEffect } from 'react';
+import {useState, formState, setFormState} from 'react';
+import {useMutation} from '@apollo/client';
+import {LOGIN_USER} from '../utils/mutations';
+import Auth from '../utils/auth';
+import { Input, Button, Box, ButtonGroup } from '@chakra-ui/react';
+import { useNavigate } from 'react-router-dom';
+// import CityRentals from '../pages/CityRentals';
+//import { ADD_LOCATION_REVIEW } from '../utils/mutations';
+import { FormControl, FormLabel, Modal, ModalOverlay, ModalContent, ModalHeader, ModalBody, ModalCloseButton, Stack } from "@chakra-ui/react"
+// import Login from './Login';
+// import Signup from './Signup';
+import { useDisclosure } from "@chakra-ui/react"
+import {Alert, AlertIcon, AlertTitle, AlertDescription, } from "@chakra-ui/react";
+import Legend from './atoms/Legend';
+import { ratingDescription } from '../utils/ratingDescriptions';
+import {Accordion, AccordionItem, AccordionButton,AccordionPanel, AccordionIcon, } from "@chakra-ui/react";
+import { Checkbox, CheckboxGroup } from "@chakra-ui/react";
+
+
+
+
+
+const NewLocationReview = ({isOpen,onClose}) => {
+
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+    const [selectedCheckbox, setSelectedCheckbox] = useState(null);
```

Comment:
we should be using radioButtons (radio's are single select") vs checkboxes (multi select component )

Reply:
Gotcha! Was trying to figure out how to make it single select.